src/components/LinkManager/LinkList/LinkOverview/LinkOverView.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+import styles from "./LinkOverview.module.scss";
+import { Link as LinkType } from "@/src/constants/types";
+import getElapsedTime from "@/src/utils/getElapsedTime";
+import getFormattedDate from "@/src/utils/getFormattedDate";
+import { LinkImageEmptyCase } from "@/src/asset/";
+import Image from "next/image";
+
+interface LinkOverview {
+  link: LinkType;
+}
+
+export default function LinkOverview({ link }: LinkOverview) {
+  const { imageSource: imageUrl, createdAt, description, title, url } = link;
+  const createDate = getElapsedTime(createdAt);
+  const formatDate = getFormattedDate(createdAt);
+
+  // 서버 Image-url이 문제가 있어서 그냥 null처리 했읍니다
+  function fixedUrl(url: string) {
+    if (!url) return null;
+
+    if (url.startsWith(`https:`)) {
```

Comment:
자바스크립트 값이 들어가지 않는 일반 문자열에는 백틱보타는 따옴표로 표현해주는게 좋습니다~
```suggestion
    if (url.startsWith('https:')) {
```

Reply:
앗 이것도 또 실수했네요 감사합니다!